kuma/attachments/utils.py

```diff
@@ -39,7 +39,7 @@ def attachment_upload_to(instance, filename):
     #
     # The md5 hash here is of the full timestamp, down to the
     # microsecond, of when the path is generated.
-    now = datetime.now()
+    now = timezone.now()
```

Comment:
This shouldn't be using the timezone.now function as it's used to build a date for a folder path. Since the timezone that timezone.now uses is request depending this may change from request to request. In other words, stick to the old datetime.now() which is using the server's timezone to build the date.